net/bridge.go

```diff
@@ -2,43 +2,126 @@ package net
 
 import (
 	"fmt"
+	"io/ioutil"
 	"net"
+	"path/filepath"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/coreos/go-iptables/iptables"
+	"github.com/j-keck/arping"
+	"github.com/pkg/errors"
 	"github.com/vishvananda/netlink"
+
+	"github.com/weaveworks/weave/common/odp"
 )
 
-type BridgeType int
+/* There are 3 modes of operation:
```

Comment:
I guess showing the `weaver` process in the doc is not that important (and might confuse a reader) as drawing the actual configuration for each setup. Maybe something along these lines:

```
There are thee configurations of bridges:

1. Bridge
                 +---------+
(container-veth)-+  weave  +-(vethwe-bridge)--(vethwe-pcap)
                 +---------+

Used by "router/pcap.go:Pcap" bridge. "vethwe-pcap" (end of veth pair) captures
and injects packets.

2. BridgedFastdp

                 +---------+                                    +------------+
(container-veth)-+  weave  +-(vethwe-bridge)--(vethwe-datapath)-+  datapath  +
                 +---------+                                    +------------+

Used by "router/fastdp.go:fastDatapathBridge" bridge. "datapath" (Open vSwitch
datapath) is connected to the "weave" bridge via veth pair.

3. Fastdp

                 +---------+
(container-veth)-+  weave  +
                 +---------+

"weave" is Open vSwitch datapath. Currently not in use due to missing conntrack
support in datapath of old kernel versions (https://github.com/weaveworks/weave/issues/1577).
```